packages/utils/src/types.ts

```diff
@@ -819,7 +836,11 @@ export type UiSchema<
   F extends FormContextType = any
 > = GenericObjectType &
   MakeUIType<UIOptionsBaseType<T, S, F>> & {
-    /** Allows the form to generate a unique prefix for the `Form`'s root prefix */
+    /** The set of Globally relevant UI Schema options that are read from the root-level UiSchema and stored in the
+     * Registry for use everywhere.
+     */
+    'ui:globalOptions'?: GlobalUISchemaOptions;
+    /** Allows the form to generate a unique prefix for the `Form`'s root prefix, deprecated in favor of  */
```

Comment:
Was this deprecated in favor of the `idPrefix` form prop? The docs say nothing about deprecation.

Reply:
Whoops, I should undo that
```suggestion
    /** Allows the form to generate a unique prefix for the `Form`'s root prefix  */
```